src/PeliasPlaceConfig.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  PeliasPlaceConfig.swift
+//  pelias-ios-sdk
+//
+//  Created by Matt Smollinger on 2/16/16.
+//  Copyright © 2016 Mapzen. All rights reserved.
+//
+
+import Foundation
+
+public struct PeliasPlaceConfig : PlaceAPIConfigData {
+  
+  var urlEndpoint = NSURL.init(string: "/v1/place", relativeToURL: PeliasSearchManager.sharedInstance.baseUrl)!
+  var queryItems = [String:NSURLQueryItem]()
+  var completionHandler: (PeliasResponse) -> Void
+  
+  var apiKey: String? {
+    didSet {
+      if let key = apiKey where apiKey?.isEmpty == false {
+        appendQueryItem("api_key", value: key)
```

Comment:
Yeah it's probably about time to add a string constants file I guess. Tagged for 1.0.0 - https://github.com/pelias/pelias-ios-sdk/issues/40